docs/concepts/what-is-filecoin.md

```diff
@@ -0,0 +1,27 @@
+---
+title: What is Filecoin?
+---
+
+# What is Filecoin?
+
+**Filecoin is a peer-to-peer network that stores files on the internet, with built-in economic incentives to ensure files are stored reliably over time.**
```

Comment:
What does "on the internet" mean? For a "normal person," the storage comparison is to centralized providers like Dropbox. I'm not sure "on the internet" sounds any different from "in the cloud" or other terms you'd use for files you upload to a centralized system. Should we switch to something that calls out the decentralized network?

Reply:
Great feedback. I have removed “on the internet” but left “peer-to-peer” as a nod to the decentralized network.